README.md

```diff
@@ -28,41 +28,46 @@ Tips on upgrading can be found in [UPGRADE.md](UPGRADE.md)
 
 ## Usage
 
-Using Herodot is straight forward, these examples will help you getting started.
+Using Herodot is straightforward. The examples below will help you get started.
 
 ### JSON
 
-Herodot supplies an interface, so it's possible to write many outputs, such as XML and others. For now, JSON is supported.
+Herodot supplies an interface, allowing to return errors in many data formats like XML and others. Currently, it supports only JSON.
 
 #### Write responses
 
 ```go
-var writer = herodot.NewJSONWriter(nil)
+var hd = herodot.NewJSONWriter(nil)
 
 func GetHandler(rw http.ResponseWriter, r *http.Request) {
-	writer.Write(rw, r, map[string]interface{}{
+	// run your business logic here
```

Comment:
I was struggling to understand these examples until later below in the error handling section I realized that these are supposed to be HTTP handlers and typically one doesn't just return a response but calls some business logic here. To make this easier, I'm adding these comments.

Reply:
Ah yeah no, it doesn't - it was broken up into several private interfaces: https://github.com/ory/herodot/blob/master/error.go#L23-L56